richreview/public/webapps/richreview/stylesheets/style.css

```diff
@@ -1058,4 +1070,25 @@ body::-webkit-scrollbar-corner {
 
 .rm_btn_center > i {
   font-size: 0.30em;
+}
+
+#r2_content_preview {
+  z-index: 10;   
+  background-color: rgba(31, 190, 242, 0.2);
+  overflow: auto;
+}
+
+#dragbar{
+   background-color:black;
+   height:100%;
+   float:  left;
+   width: 0.5%;
+   cursor: col-resize;
+}
+
+
+.r2_preview {
+  z-index: 10;
+  background-color: rgba(31, 190, 242, 0.2);
+  overflow: auto;
```

Comment:
Let's be a bit more careful naming these three elements. I'd suggest r2_preview_content, r2_preview_bar, and r2_preview_thumb.